.github/workflows/release-please.yml

```
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2023 Alliander N.V.
+#
+# SPDX-License-Identifier: Apache-2.0
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
```

```suggestion
          release-type: simple
```

For Simple types, you need to add a `version.txt` With an initial Version.

version.txt
```txt
1.0.0
```